packages/unlock-js/src/PublicLock/v9/setExpirationDuration.js

```diff
@@ -0,0 +1,17 @@
+export async function setExpirationDuration(
+  { lockAddress, expirationDuration },
+  callback
+) {
+  const lockContract = await this.getLockContract(lockAddress)
+  const transactionPromise =
+    lockContract.setExpirationDuration(expirationDuration)
```

Comment:
Maybe good to add a check for non-0 value here? Or another specific `setInfiniteDuration` function ?

Related to the support of infinite duration which currently requires `MAX_UINT`  https://github.com/unlock-protocol/unlock/issues/8222

Reply:
make sense. I will add a check to accept only non-zero or above values.